yazi-config/preset/keymap.toml

```diff
@@ -173,8 +174,9 @@ keymap = [
 	{ on = "x",       run = "cancel",  desc = "Cancel the task" },
 
 	# Help
-	{ on = "~",    run = "help", desc = "Open help" },
-	{ on = "<F1>", run = "help", desc = "Open help" },
+	{ on = "~",		run = "help", 	desc = "Open help" },
+	{ on = "<F1>",	run = "help", 	desc = "Open help" },
+	{ on = "F",		run = "filter", desc = "Apply a filter for the help items" },
```

Comment:
`[tasks]` doesn't have a command called `filter`, https://yazi-rs.github.io/docs/configuration/keymap#tasks

Reply:
How else then can you show to user that help list can be filtered when opened from tasks content?

Reply:
You need to press `~` in task manager to open its own help